tools/V4-CSDL-to-OpenAPI.xsl

```diff
@@ -897,6 +897,20 @@
 
   <!-- definitions for standard error response - only needed if there's an entity container -->
   <xsl:template match="edm:EntityContainer" mode="hashpair">
+    <xsl:choose>
+      <xsl:when test="$odata-version='2.0'">
+        <xsl:text>"@OData.metadata":{
+            "type":"object",

```

Comment:
Please do not inject line-breaks in the output, use multiple xsl:text elements if source code lines become too long